src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_stats_test.py

```diff
@@ -27,18 +26,149 @@ class PerformanceStatsTest(unittest.TestCase):
+
+  def test_parse_log_and_stats_crash(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('crash.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [],
+                                                    ['-max_len=1337'])
+    self.assertEqual(1, parsed_stats['crash_count'])
+    self.assertEqual(0, parsed_stats['corpus_crash_count'])
+    self.assertEqual(0, parsed_stats['startup_crash_count'])
+
+  def test_parse_log_and_stats_startup_crash(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('startup_crash.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [],
+                                                    ['-max_len=1337'])
+    self.assertEqual(0, parsed_stats['crash_count'])
+    self.assertEqual(0, parsed_stats['corpus_crash_count'])
+    self.assertEqual(1, parsed_stats['startup_crash_count'])
+
+  def test_parse_log_and_stats_corpus_crash(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('corpus_crash.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [], [])
+    self.assertEqual(1, parsed_stats['crash_count'])
+    self.assertEqual(1, parsed_stats['corpus_crash_count'])
+    self.assertEqual(0, parsed_stats['startup_crash_count'])
+
+  def test_parse_log_and_stats_corpus_crash_with_corpus_subset(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('corpus_crash_with_corpus_subset.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [], [])
+    self.assertEqual(1, parsed_stats['strategy_corpus_subset'])
+
+  def test_parse_log_and_stats_oom(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('oom.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [], [])
+    self.assertEqual(1, parsed_stats['oom_count'])
+
+  def test_parse_log_and_stats_oom_in_seed_corpus(self):
+    """Test stats parsing and additional performance features extraction
+    without applying of stat_overrides."""
+    log_lines = self._read_test_data('oom_in_seed_corpus.txt')
+    parsed_stats = stats.parse_performance_features(log_lines, [], [])
+    self.assertEqual(1, parsed_stats['oom_count'])
+    self.assertEqual(0, parsed_stats['timeout_count'])
+
+  def test_parse_log_and_stats_from_corrupted_output(self):
+    """Test stats parsing from a log with corrupted libFuzzer stats."""
+    log_lines = self._read_test_data('corrupted_stats.txt')
+    parsed_stats = launcher.parse_log_stats(log_lines)
```

Comment:
Why does this use parse_log_stats instead of parse_performance_features ?

Reply:
The stats being parsed here are the ones from libFuzzer, e.g. stat::peak_rss_mb.